### DISCUSSION TITLE
Add wasm compilation target

### DISCUSSION THREAD

**Discussion by Jengro777** [meilisearch/product#551] [Feedback & Feature Proposal]
For better portability, we hope to provide the deployment file of wasm

**Comment by Kerollmops** [comment]
Hey @Avey777,

Could it be possible to explain to us what's your use case?
Are you aware that Meilisearch runs on top of LMDB and that the whole engine uses the file system?
If so, do you want to use a wasi interface to make it compile correctly?

**Reply by Jengro777** [reply]
Just want to have this option, no more usage scenarios
